api/lib/domain/services/authentication/oidc-authentication-service.js

```diff
@@ -19,6 +40,76 @@ class OidcAuthenticationService {
       this.jwtOptions
     );
   }
+
+  async exchangeCodeForTokens({ code, redirectUri }) {
```

Comment:
On pourrait remettre l'ancien nom pour cette fonction, on ne récupère que l'idToken 

```suggestion
  async exchangeCodeForIdToken({ code, redirectUri }) {
```